lms/djangoapps/shoppingcart/models.py

```diff
@@ -1746,6 +1746,8 @@ def purchased_callback(self):
         self.course_enrollment.activate()
 
     def additional_instruction_text(self):
+        if not self.course_enrollment.is_verified_enrollment():
+            return ""
```

Comment:
@wedaly you mentioned you will talk to finance department about refund policy for this new course mode. Please let me know about the refund policy we will add the text according to the refund policy.